utils/jpgicc/meson.build

```
@@ -0,0 +1,24 @@
+#project('jpegicc', 'c')
+
+jpegicc_srcs = [
+  'jpgicc.c',
+  'iccjpeg.c',
+  '../common/xgetopt.c',
+  '../common/vprf.c',
+  '../common/utils.h',
+]
+
+inc_common_dirs = include_directories('../common')
+
+libjpeg_dep = dependency('libjpeg', required: get_option('jpeg'))
+if libjpeg_dep.found()
+  executable('jpegicc',
+    jpegicc_srcs,
+    c_args: cargs,
+    dependencies : [liblcms2_dep, libjpeg_dep],
+    include_directories : inc_common_dirs,
+    install : true)
+endif
+
+#man_MANS = jpgicc.1
```

The Meson way to do this is with: https://mesonbuild.com/Reference-manual_functions.html#install_man

```suggestion
install_man('jpgicc.1')
```